pkg/pullreq/comment.go

```diff
@@ -204,3 +204,31 @@ func loadTemplates() (*template.Template, error) {
 		filepath.Join(tempDir, "pkg/pullreq/templates/*.gotpl"),
 	)
 }
+
+func commentChunks(body string, maxLen int) []string {
```

Comment:
Would it be interesting to make sure that we don't split comments in the middle of a line?

Reply:
Good point, implemented.